src/Conference/UpdateConference.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import Input from "../Input";
+import styled from "styled-components";
+import { useHistory } from "react-router-dom";
+import { updateConferenceById } from "../api";
+
+const StyledConferenceForm = styled.div`
+  height: 400px;
+  padding: 20px;
+`;
+
+const StyledCardHeading = styled.h3`
```

Comment:
As you are using this headings and styled cards throughout different components it may make sense to create a new file and import these into everywhere which needs them